client/web/src/enterprise/batches/detail/bulk-operations/BulkOperationNode.tsx

```diff
@@ -0,0 +1,87 @@
+    <>
+        <div
+            className={classNames(
+                styles.bulkOperationNodeContainer,
+                'd-flex justify-content-between align-items-center'
+            )}
+        >
+            <div className="text-center p-3">
+                <p className="badge badge-secondary mb-2">{node.changesetCount}</p>
+                <p className="mb-0">{pluralize('changeset', node.changesetCount)}</p>
+            </div>
+            <div className={styles.bulkOperationNodeDivider} />
+            <div className="flex-grow-1 ml-3">
+                <h4>
+                    <CommentOutlineIcon className="icon-inline text-muted" /> Comment on changesets
+                </h4>
+                <p className="mb-0">
+                    <Link to={node.initiator.url}>{node.initiator.username}</Link> <Timestamp date={node.createdAt} />
+                </p>
+            </div>
+            {node.state === BulkOperationState.PROCESSING && (
+                <div className={classNames(styles.bulkOperationNodeProgressBar, 'flex-grow-1 ml-3')}>
+                    <meter value={node.progress} className="w-100" min={0} max={1} />
+                    <p className="text-center mb-0">{Math.ceil(node.progress * 100)}%</p>
+                </div>
+            )}
+            {node.state === BulkOperationState.FAILED && (
+                <span className="badge badge-danger text-uppercase">failed</span>
+            )}
+            {node.state === BulkOperationState.COMPLETED && (
+                <span className="badge badge-success text-uppercase">complete</span>
+            )}
+        </div>
+        {node.errors.length > 0 && (
+            <div className={classNames(styles.bulkOperationNodeErrors, 'px-4')}>
+                <Collapsible
+                    titleClassName="flex-grow-1"
+                    title={<h4 className="mb-0">The following errors occured while running this task:</h4>}
+                >
+                    {node.errors.map((error, index) => (
+                        <div className="mt-2 alert alert-danger" key={index}>
+                            <p>
+                                {error.changeset.__typename === 'HiddenExternalChangeset' ? (
+                                    <span className="text-muted">On hidden repository</span>
+                                ) : (
+                                    <>
+                                        <LinkOrSpan className="alert-link" to={error.changeset.externalURL?.url}>
+                                            {error.changeset.title} <ExternalLinkIcon className="icon-inline" />
+                                        </LinkOrSpan>{' '}
+                                        on{' '}
+                                        <Link className="alert-link" to={error.changeset.repository.url}>
+                                            repo:{error.changeset.repository.name}
```

Comment:
Since this is human readable, should we go with:

```suggestion
                                            repository {error.changeset.repository.name}
```

Reply:
Can do. I think we were trying to save some space in the design, but it probably doesn't matter too much